single_steps/salmon_index.smk

```diff
@@ -16,6 +16,7 @@ rule salmon_index:
         directory(output_index_dir)
     params:
         threads = threads
+    conda: "../envs/single_steps.yaml"
     shell:
         """
         /SAN/vyplab/alb_projects/tools/salmon-latest_linux_x86_64/bin/salmon index \
```

Comment:
We already have `single_steps/salmon.smk` that is much cleaner than this (uses conda etc.) and more flexible. We can get that to only build the index by passing `--until salmon_index`. Guess only difference is that this one won't include decoy sequences, which generally isn't a recommendation unless you want this for backwards compatibility?

Also doesn't matter if you add the `conda` directive it'll still use the salmon in your tools dir